ivy/functional/backends/torch/experimental/manipulation.py

```diff
@@ -328,3 +330,105 @@ def concat_from_sequence(
     elif new_axis == 1:
         ret = torch.stack(input_sequence, dim=axis)
         return ret
+
+
+def _can_handle_padding(input_shape, pad, mode):
+    n_dims = len(input_shape)
+    if n_dims == 0:
+        return False
+    if n_dims == 1 and mode != 'circular':
+        return False
+    if mode == 'circular':
+        return True
+    if isinstance(pad, int):
+        return False
+    for i in range(n_dims):
+        if not isinstance(pad[2 * i], int) or not isinstance(pad[2 * i + 1], int):
+            return False
+        if input_shape[i] + pad[2*i] + pad[2*i+1] < 0:
+            return False
+    return True
+
+
+def _check_tuple(t):
+    if isinstance(t, tuple):
+        if len(t) == 1 and not isinstance(t[0], tuple):
+            return t[0], True
+        else:
+            has_multiple_values = False
+            size = 0
+            for elem in t:
+                elem_val, elem_has_multiple_values = _check_tuple(elem)
+                if elem_has_multiple_values:
+                    has_multiple_values = True
+                size += elem_val
+            return size, has_multiple_values
+    else:
+        return 1, False
+
+
+def _check(*args, **kwargs):
```

Comment:
this logic seems to be duplicated. Is it possible to refactor it maybe in a helper file?

Reply:
to reduce the redundancy, I imported the helper function _check_tuple in TensorFlow backend from PyTorch backend